activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   When calling `Relation#sum` `on a has_many_through` relation, check if distinct is true and set the arel object
```

Comment:
I think a better changelog entry would be the title of this PR:

"Ensure `sum` honors `distinct` on `has_many :through` associations"